belt and suspenders : don't mess up with chromium, use .config/electron

Index: chrome/common/chrome_paths_linux.cc
--- chrome/common/chrome_paths_linux.cc.orig
+++ chrome/common/chrome_paths_linux.cc
@@ -89,13 +89,7 @@ bool GetDefaultUserDataDirectory(base::FilePath* resul
         GetXDGDirectory(env.get(), kXdgConfigHomeEnvVar, kDotConfigDir);
   }
 
-#if BUILDFLAG(GOOGLE_CHROME_FOR_TESTING_BRANDING)
-  std::string data_dir_basename = "google-chrome-for-testing";
-#elif BUILDFLAG(GOOGLE_CHROME_BRANDING)
-  std::string data_dir_basename = "google-chrome";
-#else
-  std::string data_dir_basename = "chromium";
-#endif
+  std::string data_dir_basename = "electron";
   *result = config_dir.Append(data_dir_basename + GetChannelSuffixForDataDir());
   return true;
 }
